services/storage/tests/unit/test_rpc_handlers_paths.py

```diff
@@ -93,6 +96,39 @@ async def _assert_compute_path_size(
     pytest.fail("Job did not finish")
 
 
+async def _assert_delete_paths(
+    storage_rpc_client: RabbitMQRPCClient,
+    location_id: LocationID,
+    user_id: UserID,
+    product_name: ProductName,
+    *,
+    paths: set[Path],
+) -> None:
+    async_job, async_job_name = await delete_paths(
+        storage_rpc_client,
+        product_name=product_name,
+        user_id=user_id,
+        location_id=location_id,
+        paths=paths,
+    )
+    await asyncio.sleep(1)
```

Comment:
please use tenacity here, instead of having a fixed sleep, thee always end up being flaky

Reply:
I will try to remove it. but I think that is a "feature" of our implementation.